src/core/server/handler/handler.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <iostream>
+#include <fstream>
+#include <set>
+#include <vector>
+
+#include <cpprest/http_listener.h>
+#include <cpprest/json.h>
+#include <cpprest/uri.h>
+#include <cpprest/asyncrt_utils.h>
+#include <cpprest/json.h>
+#include <cpprest/filestream.h>
+#include <cpprest/containerstream.h>
+#include <cpprest/producerconsumerstream.h>
+
+#define TRACE(msg)            ucout << msg
+#define TRACE_ACTION(a, k, v) ucout << (a) << " (" << (k) << ", " << (v) << ")\n"
```

Comment:
I do not like these macroses. 

Please consider using `static constexpr` functions.

Reply:
They are only for test purposes